contents/uix/accessibility.md

```diff
@@ -0,0 +1,181 @@
+
+## How can I improve the accessibility of my website?
+
+There are many ways to improve the accessibility of your website. Below are some tips, based on the type of disabilities they address.
+
+### Visual
+Users with visual impairments rely on assistive technologies such as a magnifier or a screen reader.
+
+##### :fas-thumbs-up: Best practice: Use more visual indicators to convey a message
+  
+Let us look at the Facebook sign up page. Suppose I want to sign up for a new account, but I have not put in all the necessary information. So, it is supposed to tell me that my attempt to sign up has failed.
+
+This page below uses a red color border around the text box to show that the information needed is missing. As red usually signifies failure, this seems enough.
+![Facebook sign up page](accessibility/fb-normal.png "Facebook sign up page")
+
+However, to someone with a type of color blindness called [achromatopsia](https://en.wikipedia.org/wiki/Achromatopsia), this is how the page looks like:
+![Facebook sign up page to someone with achromatopsia](accessibility/fb-achromatopsia.png "Facebook sign up page to someone with achromatopsia")
+
+To someone with another type of color blindness, [deuteranopia](https://en.wikipedia.org/wiki/Deuteranopia), the page looks like this:
+![Facebook sign up page to someone with deuteranopia](accessibility/fb-deuteranopia.png "Facebook sign up page to someone with deuteranopia")
+
+
+It may be unclear to color-blind users that the sign up has failed. To avoid confusion, we should not rely on colours alone to convey a message. Instead, we should use more visual indicators such as icons or an explanation box.
+![Sign up page with more visual indicators](accessibility/fb-more-indicators.png "Facebook sign up page with more visual indicators")
+
+##### :fas-thumbs-up: Best practice: Maintain good contrast
+  
+Look at this page taken from Tech Crunch.
+![Tech Crunch page](accessibility/tc-normal.png "Tech Crunch page")
+
+To users with good vision, the design may look minimalist and clean. However, to someone suffering from cataract, the page looks like this.
+
+![Blurry Tech Crunch page](accessibility/tc-blurred.png "Tech Crunch page to someone with cataract")
+It gets difficult to read the news snippets because of the poor color contrast (grey on white). On the other hand, the black colored texts are still legible. Thus, we should maintain good contrast ratio in our websites for ease of reading.
+
+##### :fas-thumbs-up: Best practice: Use `alt` attribute for images
+  
+The `alt` attribute provides alternative information for an image that can be read out by a screen reader to describe an image.
+
+![Use of alt tag](accessibility/alt-tag.png "Alt tag")
+
+The above are some examples of best practices to follow when developing for web accessibility. More can be found in [this article on developing for accessibility](https://www.w3.org/WAI/tips/developing/).
```

Comment:
Are these best practices to follow for developing for web accessibility in general or just for visual accessibility?